pkg/kobject/kobject.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 var unsupportedKey = map[string]int{
-	"Build":         0,
+	// "Build":         0,
```

Comment:
We need to have provider based checks for unsupported keys, as, ``build`` is now supported for ``openshift`` but not for ``kubernetes``.